GiniVision/Tests/CameraButtonsViewControllerTests.swift

```diff
@@ -8,6 +8,7 @@
 import XCTest
 @testable import GiniVision
 
+// swiftlint:disable type_body_length
```

Comment:
You can consider splitting the tests rather than having a linter exception (if you want).

Reply:
I think it's fine like this